crypto/objects/obj_dat.h

```diff
@@ -2,7 +2,7 @@
  * WARNING: do not edit!
  * Generated by crypto/objects/obj_dat.pl
  *
- * Copyright 1995-2022 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
The year is reverse ;)

Reply:
These are fixed by the release scripts.  There is no need to manually do this change.